Doc/library/mmap.rst

```diff
@@ -149,16 +149,24 @@ To map anonymous memory, -1 should be passed as the fileno along with the length
       import mmap
       import os
 
-      mm = mmap.mmap(-1, 13)
-      mm.write(b"Hello world!")
-
-      pid = os.fork()
-
-      if pid == 0:  # In a child process
-          mm.seek(0)
-          print(mm.readline())
-
-          mm.close()
+      HELLO = b"Hello World!\n"
+      HOLA  = b"Hola Munda!\n"
```

Comment:
Is this supposed to be in Spanish? If so, I think the correct sentence would be `¡Hola Mundo!`

Reply:
ok thanks ill fix that

Reply:
well actually '¡' being non ASCII, that wouldn't fit in a byte. I''ll just change it to something English.

Reply:
OK  I changed it to just write process ids to the shared memory. And got rid of unnecessary comments. What do you think?
It shows that a process can open the buffer, write to it, fork, child can read and write, and that the changes are visible to parent process.